packages/compass/src/app/components/update-toasts.tsx

```diff
@@ -53,6 +53,7 @@ const RestartCompassToastContent = ({
       <button
         className={cx(buttonStyles, darkmode && buttonDarkStyles)}
         onClick={onUpdateClicked}
+        data-testid="auto-update-restart-button"
```

Comment:
As a drive-by I'm adding these test ids so the latest nightly dev release will end up with them and then the auto-update tests can use them.